### PR TITLE
Networking Jason

### PR DESCRIPTION
**Pull request by JasonStallkamp**
Added basic Networking to game mechanics version for 0.0.0.1

### CODE REVIEW

**Comment by JasonStallkamp** [comment]
[LAN Networking](https://trello.com/c/UtyEzX0o/11-lan-networking)

File: Local War/Assets/Script/Weapon/Bullet/LaserAmmo.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        damage = 15.0f;
+
+        //If put the get function in Start then it will not execute properly
+        line = GetComponent<LineRenderer>();
+
+        destination = direction;
+        line.SetPosition(0, origin);
+        line.SetPosition(1, destination);
+
+        if (!isServer)
```
Comment (Alasses) [inline Local War/Assets/Script/Weapon/Bullet/LaserAmmo.cs:24]: This is an important logical point. (Need to discuss)

File: Local War/Assets/Script/Weapon/RangeWeapon.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    private int     magzine;
+    private int     totalAmmo;
+    private float   coolDown;
+    private float   coolDownCounter;
+    private bool    isShooting;
+
+    public GameObject ammoType;
+    public GameObject firePoint;
+
+    public void Fire(bool isServer)
```
Comment (Alasses) [inline Local War/Assets/Script/Weapon/RangeWeapon.cs:17]: Important logical point. (Need discuss)

**Review by Alasses — APPROVED** [review]
Everything looks good, but the logic of the weapon system needs to be discussed.

**Pull request merged**